src/otx/core/data/module.py

```diff
@@ -103,6 +105,17 @@ def _get_dataset(self, subset: str) -> OTXDataset:
             raise KeyError(msg)
         return dataset
 
+    def _postprocess(self, dataset: Dataset) -> Dataset:
+        """Post processing after importing dataset from Datumaro."""
+        if self.task == "DETECTION":
```

Comment:
It seems that we don't need to add condition `self.task == "DETECTION" in the current status. Maybe all other tasks should have items that have more than 1 annotations.

Reply:
Done by [6181efd](https://github.com/openvinotoolkit/training_extensions/pull/2823/commits/6181efd111b4b8eecd781ab9cf78eda2fc40d87c)